### ISSUE TITLE
Import into Quicken

### ISSUE THREAD

**Issue by jefftb** [fcfort/betterment-csv-chrome#68]
Will this plug-in work to import the QIF or CSV into Quicken?

**Comment by fcfort** [comment]
Yep, that's exactly what this extension is for. Clicking the extension icon in Chrome will let you choose between output formats (either QIF or CSV).

**Comment by jefftb** [comment]
I’ve not had success. My goal would be to import the statement activity into my existing betterment account in quicken. Quicken for Mac won’t import either the csv or qif.

**Comment by fcfort** [comment]
Hmm seems like the latest versions of Quicken for Mac don't support the QIF format: https://community.quicken.com/discussion/7857018/need-the-ability-to-import-qif-transactions. 

Any suggestions for what will work for you? I don't have access to a copy of the software.

**Comment by jefftb** [comment]
Ah, I didn't realize that either. QFX is what has worked well from my other banks to import transactions. 

I am attaching a screenshot of the import screen from Quicken for Mac:  

<img width="296" alt="3" src="https://user-images.githubusercontent.com/17056545/87053910-bbafdb00-c1bf-11ea-8037-08b476db95d2.png">
